Eastern-Research-Group/mywaterway
# Feature/726 more climate tab changes

## Related Issues:
* [HMW-726](https://jira.epa.gov/browse/HMW-726)

## Main Changes:
* Added a `Clear Selections` button above the radio button section.
* Added more text to the extreme weather tab.
* Updated the styles of the providers layer for the extreme weather tab. Extreme weather tab is a black outline and all other tabs are the yellow outline.  
* For the Hazard Potential Index field in the dams popup, added a symbol and info modal. The info modal only shows the explanation for the current hazard index.
* Added more fields to the CSO popups. Not all of the discharger properties were available in the CSO layer.
* Added more fields to the storage tanks popups.  
* Added mention of grts success stories on restore tab to the data page.
* Changed the layer title from `Who provides the drinking  water here?` to `Providers` for consistency with the legend.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/extreme-weather
2. Verify the county outline is black (check map and legend)
3. Verify the providers layer is titled as "Selected County" (check legend and layer list)
4. Switch to the drinking water tab
5. Verify the county outline is yellow (check map and legend)
6. Verify the providers layer is titled as "Providers" (check legend and layer list)
7. Switch to the extreme weather tab
8. Verify the text matches the word document on the ticket (two sections one at the top of the page and the other at the bottom)
9. Turn on one of the radio buttons
10. Click `Clear Selections` and verify all radio buttons are off
11. Turn on the `Above ...storage tanks` layer
12. Verify the popup has all of the fields mentioned in the email chain (in comment on ticket)
13. Turn on the `Combined Sewer Overflows` layer
14. Verify more fields have been added (Note: the layer we are using doesn't have all of the fields in echo, so I maded a TODO for this one)
15. Turn on the `Dams` layer
16. Verify the popup has all of the fields mentioned in the email chain (in comment on ticket)
17. Verify the `Hazard Potential Index` field has a symbol and info icon next to it
18. Click the info icon
19. Verify it has explanatory text for the hazard potential index value
21. Click the info icon next to the "Dams" layer
22. Verify the modal has explanations for the 3 main values (high, significant and low)
23. Open the data page
24. Verify that under GRTS, the success stories sub-tab is mentioned.

## TODO:
- [ ] May need to rework the CSO layer, if we can find a query to get them from echo like dischargers.
